### ISSUE TITLE
Halves fps on big modpack

### ISSUE THREAD

**Issue by verklig** [Buuz135/DarkModeEverywhere#36]
Hey, I was trying to figure out what was tanking my fps for a while and found out that this was taking it down from around 150-160 to 70-80 fps when using any other mode than light mode (normal).

I first noticed it when I pressed f1 to hide the ui and saw the fps increase immediately. I am playing the ATM8 modpack if that helps,

Just a shame because it does look really good!

**Comment by Lordfirespeed** [comment]
Which version of the mod are you using?

**Comment by Lordfirespeed** [comment]
Assuming 1.1.2 (the version listed in the modpack change history), that's really odd

**Comment by verklig** [comment]
Yeah im playing the latest version of ATM8 which is 1.0.18 as of now, and I just checked the mod folder and it says version 1.1.2 as you stated.

I also confirmed that other friends are having the same issue so it doesn’t seem to be on our end.

**Comment by Lordfirespeed** [comment]
I have an idea of what it might be and I'm looking into solutions ❤️

**Comment by verklig** [comment]
No there was no connection to any block or ui, that's why it was so hard to find what was causing it. Are you using f3 to show fps or do you use something else? The better f3 mod lowers the fps too so i check it with shadowplay with fps counter from there.

I can provide a video if you'd like to see how big the difference is, if so I would just need to find something else to show the fps with.

**Comment by LazerguiY** [comment]
I'm experiencing the same issue on my end on latest ATM8 (mod version 1.1.2). The HUD itself is dropping FPS without having any GUI open.

**Comment by Buuz135** [comment]
Might be fixed in the latest versions
